go-controller/pkg/ovn/namespace.go

```diff
@@ -196,6 +198,10 @@ func (oc *Controller) AddNamespace(ns *kapi.Namespace) {
 			klog.Errorf(err.Error())
 		}
 	}
+	annotation = ns.Annotations[aclLoggingAnnotation]
+	if annotation != "" && oc.aclLoggingCanEnable(annotation, nsInfo) {
+		klog.Infof("Namespace %s: ACL logging is set to deny=%s allow=%s", ns.Name, nsInfo.aclLogging.Deny, nsInfo.aclLogging.Allow)
+	}
```

Comment:
I would add a warning here if we failed to enable ACL logging for a namespace if user `fat fingered` the severity level names. So, I would do

```
if annotation != "" {
    if oc.aclLoggingCanEnable() {
      // log success
    } else {
      // log failure warning
   }
}
```